src/Symfony/Component/TypeInfo/Tests/Type/CollectionTypeTest.php

```diff
@@ -119,9 +119,6 @@ public function testAccepts()
 
         $this->assertTrue($type->accepts(new \ArrayObject([1 => true])));
         $this->assertFalse($type->accepts(new \ArrayObject(['foo' => true])));
-
-        $type = new CollectionType(Type::generic(Type::builtin(TypeIdentifier::ITERABLE), Type::int(), Type::bool()));
```

Comment:
I was a bit confused when reading this test added in #59291 as I thought this would be a new test case but it actually seems to be the same as defined on line 118 /cc @mtarld